public/data/abhinavkashyap061.json

```
@@ -0,0 +1,28 @@
+{
+    "name": "Abhinav Kashyap",
+    "type": "personal",
+    "bio": "Full Stack Web Developer",
+    "avatar": "https://github.com/abhinavkashyap061.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/abhinavkashyap061",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/abhinavstwt",
+        "icon": "twitter"
+      },
+      {
+        "name": "Visit my Portfolio website",
+        "url": "https://abhinavkashyap061.github.io/",
+        "icon": "website"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/abhinavkashyap061/",
+        "icon": "linkedin"
+      }
+    ]
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```